app/SearchApi/BcGov.Fams3.SearchApi.Contracts/PersonSearch/ProviderProfile.cs

```diff
@@ -6,5 +6,6 @@
     public interface ProviderProfile
     { 
         string Name { get; }
+        DataProviderEnums DataProviderID { get;  }
```

Comment:
Why do we have both? Shouldn't it be just the Provider Name (the string value of the enum?)

Reply:
I can remove Name, only leave 
DataProviders DataProviderID{get;}

Reply:
Okay.. Can we remove the ID cos it's an enum so It can be DataProvider DataProvider {get;}

Reply:
ok